### PR TITLE
Feature/PC console messaging

### PR DESCRIPTION
**Pull request by zbialecki**
Add messaging buttons to Area Chair and Reviewer tabs of the PC console.

Screen Shot 2020-03-09 at 1 03 27 PM

Screen Shot 2020-03-09 at 1 25 31 PM

### CODE REVIEW

**Comment by zbialecki** [comment]
You can now set the from email via the `EMAIL_SENDER` constant. For example, 

`var EMAIL_SENDER = { name: 'ECCV OpenReview', email: 'eccv2020@openreview.net' };`

**Comment by melisabok** [comment]
Can I see the count of reviewers/Ac I'm messaging?

Screen Shot 2020-03-11 at 1 06 47 PM

File: openreview/conference/templates/programchairWebfield.js [hunk 1]
```diff
@@ -18,6 +18,7 @@ var REVIEWERS_ID = '';
 var AREA_CHAIRS_ID = '';
 var PROGRAM_CHAIRS_ID = '';
 var REQUEST_FORM_ID = '';
+var EMAIL_SENDER = null;
```
Comment (melisabok) [inline openreview/conference/templates/programchairWebfield.js]: can the default value be an `{}`

**Comment by melisabok** [comment]
I'm seeing you are sending a message per AC, you can use a single request and pass all the ACs in the `groups` parameter:

Screen Shot 2020-03-11 at 1 10 40 PM

**Comment by melisabok** [comment]
We should a single request call to send the messages. Multiple calls don't work for reviewers:

Screen Shot 2020-03-11 at 1 30 16 PM

**Comment by zbialecki** [comment]
OK, I'll make those changes.

**Comment by zbialecki** [comment]
OK, both those issues should be fixed now.

**Comment by melisabok** [comment]
I'm getting this error:

Screen Shot 2020-03-11 at 2 59 43 PM

It seems the localStorage is saving a json with groups that you are trying to iterate here.

**Pull request merged**